### PR TITLE
Subheadings to have anchor links

### PR DESCRIPTION
**Pull request by hngerebara**
Add anchor links to sub headings

Before:
Screenshot 2019-09-15 at 2 15 00 AM

After:
Screenshot 2019-09-15 at 2 14 48 AM

### CODE REVIEW

**Comment by Etiene** [comment]
Thanks for the fix!
Some questions:

1. Is changing colors back to black on focus intentional?
2. It looks slightly misaligned at the bottom. Is that also intentional? Experimenting on it, giving it a 0.9em font size made it look more aligned. What do you think?

**Comment by hngerebara** [comment]
1. According to Sam, the subheading test should not have any color on hover.
2. I'm not sure I understand this.
> It looks slightly misaligned at the bottom. Is that also intentional? Experimenting on it, giving it a 0.9em font size made it look more aligned. What do you think?

 What are you looking at?

**Comment by Etiene** [comment]
> 1. According to Sam, the subheading test should not have any color on hover.

I didnt mean the heading, I mean the icon for the anchor link. It gets blue on hover, but goes back black on focus, this gives a weird flickering effect. I think this focus is inherited from somewhere else, so it might be a little hard to find.

> 2. I'm not sure I understand this. What are you looking at?

The screenshot you just posted and the branch on my local dev. The anchor link icon is slighly too low
64915439-baa84b80-d75e-11e9-976b-78c5d6c550b3

**Comment by hngerebara** [comment]
FIxed this. You can view [here](https://drive.google.com/file/d/1d6jvsdu7nF2MNRqjTH-IPTb3yA6YxA38/view?usp=sharing)

File: assets/css/global.less [hunk 1]
```diff
@@ -1970,6 +1970,13 @@ body .guides .guides-section-white {
   }
   h1, h2, h3, h4, h5, h6 {
     color: @guides-font;
+    margin: 10px 0;
```
Comment (brikis98) [inline assets/css/global.less]: Why add a margin on top and bottom?

File: assets/css/global.less [hunk 2]
```diff
@@ -1970,6 +1970,13 @@ body .guides .guides-section-white {
   }
   h1, h2, h3, h4, h5, h6 {
     color: @guides-font;
+    margin: 10px 0; 
+    &:hover > a {
```
Comment (brikis98) [inline assets/css/global.less]: Where does the `a` come from? Is that added automatically by JS?

**Comment by Etiene** [comment]
The alignment looks good on your video, but it's still off on my screen when I check out the branch. Are there any changes you didn't push?

File: assets/css/global.less (hunk 1 above)
Comment (hngerebara) [reply assets/css/global.less]: The mockup has some margin for all header tags.

File: assets/css/global.less (hunk 2 above)
Comment (hngerebara) [reply assets/css/global.less]: Jekyll adds an anchor tag to all headers

**Comment by hngerebara** [comment]
@Etiene I still cannot replicate this issue and this branch is updated. Maybe we should quickly have a sync

File: assets/css/global.less (hunk 1 above)
Comment (brikis98) [reply assets/css/global.less]: Yes, but I believe the spacing for headers looks correct already in master, after a bunch of tweaks from yesterday. Can you show what change this is making?

**Pull request merged**